web/cobrands/fixmystreet-uk-councils/alloy.js

```diff
@@ -169,8 +167,9 @@ OpenLayers.Format.AlloyV2 = OpenLayers.Class(OpenLayers.Format.GeoJSON, {
 
 OpenLayers.Protocol.AlloyV2 = OpenLayers.Class(OpenLayers.Protocol.Alloy, {
     tileSize: 128,
+    resolution: 4.777314267158508,
```

Comment:
Does this affect Hackney's use?

Reply:
Yes in as much as Hackney is likely to not work without this.